docs/src/tutorials/tutorial_5.md

```diff
@@ -0,0 +1,377 @@
+   discovered by: Carl Wilhelm Scheele
+        named by: Antoine Lavoisier
+          source: https://en.wikipedia.org/wiki/Oxygen
+  spectral image: https://en.wikipedia.org/wiki/File:Oxygen_spectre.jpg
+```
+
+As fellow Julian, Johann-Tobias Schäg, said, one should learn `Unitful.jl` if they want to interact with the real world.
+`Unitful.jl` handles physical quantites such as pounds, meters, mols, etc. with minimal overhead in Julia.
+Further, it helps one to keep track of units and easily convert between different measurement systems.
+
+## Setting Up Our Animation
+
+As always, let's import our needed packages:
+
+```julia
+using Javis
+using PeriodicTable
+using Unitful
+```
+
+And let's define our background function:
+
+```julia
+function ground(args...)
+    background("white")
+    sethue("black")
+end
+```
+
+Finally, let's get started with creating our `javis` function:
+
+```julia
+demo = Video(500, 500)
+javis(
+    demo,
+    [
+        BackgroundAction(1:500, ground),
+	...
+    ],
+    pathname = "atomic.gif",
+    framerate = 10,
+)
+```
+
+As you can see, the animation we are creating is going to have many frames!
+This is the longest animation we have made so far.
+Why?
+Not only are we going to examine many different elements, this tutorial also serves to illustrate how one can make longer animations to convey ideas.
+Think of it as your directoral debut! 🎬 🎥
+
+## Taming the Elements!
+
+Each element has a different atomic mass.
+This atomic mass is measured in the unit called a "Dalton" (symbol: u) which is equivalent to 1/12 of the mass of a stationary carbon-12 atom.
+We can use the `Scaling` functionality that `Javis.jl` provides to visualize these different masses!
+
+To accomplish this, we need to make a function the shows our currently viewed element:
+
+```julia
+function element(p = O, color = "black")
+    sethue(color)
+    c = circle(p, 1, :fill)
+end
+```
+
+Essentially, all the `element` function does is create a circle in the middle of the frame with the initial size of 1.
+
+From there, we need to define more `Action` objects for our `javis` function for the element we are viewing to scale:
+
+```julia
+...
+        Action(
+            (args...) -> element(),
+            subactions = [
+                SubAction(1:300, Scaling(1, 30)),
+                SubAction(301:350, Scaling(30, 16)),
+            ],
+        ),
+	...
+        pathname = "atomic.gif",
+        framerate = 10
+    )
+...
+```
+
+This will grow our element from `1` to `30` and then shrink the element from `30` to `16`.
+
+That scaling looks like this:
+
+![](assets/blank_atom_scaling.gif)
```

Comment:
this shows just a black dot for me.
**Edit:** It shows a black dot for me when I look at it in the last 150 frames. I was just confused.

Reply:
This has been adjusted!

Reply:
Do you mind to change the framerate only for this one to 30? Maybe even in general. 50s is a long time for people on the internet (we don't have that much time :smile: )

Reply:
But it's okay don't want that you have to do everything over again. Just a general comment.

Reply:
Fixed this. :)